test/test_invalid_headers.py

```diff
@@ -282,7 +285,12 @@ def test_headers_event_skipping_validation(self, frame_factory, headers):
         c.clear_outbound_data_buffer()
         c.send_headers(1, headers)
 
-    @pytest.mark.parametrize('headers', invalid_header_blocks)
+        # Ensure headers are still normalized.
+        norm_headers = h2.utilities.normalize_outbound_headers(headers, None)
```

Comment:
None of the `normalize_outbound_headers` methods currently use `HeaderValidationFlags` but I also feel like passing `None` is asking for trouble later. I have a commit adding this ready to push, but it feels messy for what it's providing. Is there a better way to retrieve the `HeaderValidationFlags` for this given event, or do we need to build it from scratch?
```python
        hvf = h2.utilities.HeaderValidationFlags(
            is_client=True,
            is_trailer=False,
            is_response_header=False,
            is_push_promise=False
        )
        norm_headers = h2.utilities.normalize_outbound_headers(headers, hvf)
```